src/utils/validationSchemaSignUp.js

```diff
@@ -0,0 +1,51 @@
+import * as Yup from "yup";
+
+const schema = Yup.object().shape({
+    firstName: Yup.string()
+        .matches(/^[\u0621-\u064A\u0660-\u0669 a-zA-Z\s]+$/, "letterSpace")
+        .required("required"),
+    lastName: Yup.string()
+        .matches(/^[\u0621-\u064A\u0660-\u0669 a-zA-Z\s]+$/, "letterSpace")
+        .required("required"),
+    email: Yup.string()
+        .email("invalidEmailFormat")
+        .matches(/^[^@]+@[^@]+\.[a-zA-Z]{2,}$/, "invalidEmailFormat")
+        .required("required"),
+
+    confirmEmail: Yup.string()
+        .oneOf([Yup.ref("email"), null], "emailsMustMatch")
```

Comment:
- This code returns the word `emailsMustMatch` where I should get the localized value.
![image](https://user-images.githubusercontent.com/30384731/231485919-818a306c-6462-40d3-9908-5b5115508f8a.png)

![image](https://user-images.githubusercontent.com/30384731/231486173-3f3c015e-610a-4dec-95cc-39727f6538d4.png)